zerver/webhooks/jira/view.py

```diff
@@ -227,24 +227,29 @@ def handle_created_issue_event(payload: Dict[str, Any]) -> str:
 def handle_deleted_issue_event(payload: Dict[str, Any]) -> str:
     return u"{} **deleted** {}!".format(get_issue_author(payload), get_issue_string(payload))
 
+JIRA_CONTENT_FUNCTION_MAPPER = {
+    "jira:issue_created": handle_created_issue_event,
+    "jira:issue_deleted": handle_deleted_issue_event,
+    "jira:issue_updated": handle_updated_issue_event,
+    "comment_created": handle_updated_issue_event
```

Comment:
You can use `functools.partial` to eliminate the `if` statements below. See [this part](https://github.com/zulip/zulip/blob/master/zerver/webhooks/clubhouse/view.py#L475) of our Clubhouse integration to see how to do it!

Here's the rationale behind this: we need a uniform way of mapping events types to their respective functions in all of our webhooks because that will help us down the road when we add feature for turning off specific webhook events in our bot-creation UI. :)